src/firefly/js/Firefly.js

```diff
@@ -182,7 +182,7 @@ const defFireflyOptions = {
         canCreateExtractionTable: false,
     },
     tapObsCore: {
-        enableObsCoreDownload: true,
+        enableObsCoreDownload: true
```

Comment:
how about `enableObsCoreAttributesQuery`. The name is not quite clear.

Reply:
Oh, I didn't mean to change it (I accidentally removed comma), the prop is in SUIT.js called `tapObsCore.enableMetadataLoad` (L176)